module/zfs/spa_misc.c

```diff
@@ -2689,5 +2689,9 @@ MODULE_PARM_DESC(zfs_ddt_data_is_special,
 module_param(zfs_user_indirect_is_special, int, 0644);
 MODULE_PARM_DESC(zfs_user_indirect_is_special,
 	"Place user data indirect blocks into the special class");
+
+module_param(zfs_special_class_metadata_reserve_pct, int, 0644);
+MODULE_PARM_DESC(zfs_special_class_metadata_reserve_pct,
+	"Small blocks in special blocks depends on this much free space available");
```

Comment:
"Small blocks in special ~blocks~ vdevs depend on this much free space available"

Reply:
"Small **file** blocks in special ~blocks~ vdevs depend on this much free space available"

"Small file blocks" is the language (already in use) that I standardized on in  #8713.